src/main/java/org/polypheny/jdbc/types/ExtraPolyTypes.java

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-package org.polypheny.jdbc;
+package org.polypheny.jdbc.types;
 
+public interface ExtraPolyTypes {
 
-public interface PolyphenyJdbcStatement extends java.sql.Statement {
+    int GEOMETRY = 2015; // TODO: confirm
```

Comment:
What are extra poly types?

Reply:
This was used in a single map for type lookup. I rebuilt this part, making this interface obsolete and removed it.